### ISSUE TITLE
[newman_vocoded_2012] Update import script to match 2024 schema

### ISSUE THREAD

**Issue by adriansteffan** [langcog/peekbank-data-import#67]
No aux column, there seems to be no cdi data on osf

**Comment by adriansteffan** [comment]
The raw data is a bit messy here, the import script seems fragmented. This might need a closer look

**Comment by mzettersten** [comment]
Yes, that one never made it fully through import. These datasets are definitely trickier so let's prioritize other datasets before circling back.
